tests/tests.rs

```diff
@@ -466,3 +466,47 @@ fn dump_preprocessed_input() {
         "cpp-empty-layout.hpp is in the preprocessed file"
     );
 }
+
+
+#[test]
+fn test_array_pointers() {
+    let actual = builder()
```

Comment:
Instead of this, you can add `bindgen-flags: --use-array-pointers` to the header, which is much more maintainable. These tests should just be for options that cannot be tested using the regular cli flags or such.

Reply:
Done